tests/pipeline/test_factor.py

```diff
@@ -43,6 +48,20 @@ class F(Factor):
     window_length = 0
 
 
+class C(Classifier):
+    dtype = int64_dtype
+    missing_value = -1
+    inputs = ()
+    window_length = 0
+
+
+class OtherC(Classifier):
+    dtype = int64_dtype
+    missing_value = -1
+    inputs = ()
+    window_length = 0
```

Comment:
It's used in `test_normalizations` below for a case where I needed two different classifiers.